runtime/Cargo.toml

```diff
@@ -12,6 +12,7 @@ serde = { version = "1.0.101", default-features = false, optional = true, featur
 parity-scale-codec = { version = "2.0.0", default-features = false, features = ["derive"] }
 log = "0.4"
 hex-literal = "0.3.1"
+sha3 = { version = "0.8", default-features = false }
```

Comment:
0.9 is out now, shall we use it?

Reply:
Actually, do we really use this? I see we import some stuff from it, but I don't see us using it.

Reply:
Yep, we use Keccak256. Bumped to 0.9.